docs/_posts/2017-12-15-jekyll-3-7-0-released.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: "Jekyll 3.7.0 Released"
+description: "Jekyll 3.7.0 brings livereload, ."
+date: 2017-12-15 12:31:40 +0100
+author: dirtyf
+version: 3.7.0
+categories: [release]
+---
+
+We're happy to announce that Jekyll latest minor release is out before Christmas!
+
+Here are a few of the latest gifts 🎁 from our contributors:
+
+ * LiveReload is available as an option during development: `bundle exec jekyll server --livereload`, no more manual page refresh. A big thanks to @awood for this feature and to @andreyvit, livereload author.
```

Comment:
`bundle exec jekyll serve --livereload`
We've moved away from the `server` command since long..

:+1: for crediting the JS author